<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coffee100">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cola200">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cola(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>250</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tea150">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
